scripts/zones/FeiYin/mobs/Mind_Hoarder.lua

```diff
@@ -5,6 +5,31 @@
 ---@type TMobEntity
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onMobWeaponSkillPrepare = function(mob, target)
+    -- mind hoarder only uses mind drain
+    -- therefore override the normal list
+    return xi.mobSkill.MIND_DRAIN
+end
```

Comment:
I am missing something. If the mob only uses 1 skill no matter what, why not have their list only have the one skill they can use?

Reply:
> I am missing something. If the mob only uses 1 skill no matter what, why not have their list only have the one skill they can use?

They do not have their own skill list currently as they share a list with generic pots, thus this is alternative to creating a new list just for mind hoarder. I can do that instead if it would be better.

Reply:
Yes, I would definately give it its own list, instead of coding an exception.

Maybe in a future, we will be able to define skill lists in the mob-script directly and be done with that sql, but for now...

Reply:
> Yes, I would definately give it its own list, instead of coding an exception.
> 
> Maybe in a future, we will be able to define skill lists in the mob-script directly and be done with that sql, but for now...

Yep, I updated the PR to add its own list, thanks!